target/linux/ramips/dts/ApexeraNetBot.dts

```diff
@@ -0,0 +1,183 @@
+/dts-v1/;
+
+#include "mt7620a.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "apexera-netbot", "ralink,mt7620a-soc";
```

Comment:
`compatible = "apexera,netbot", "ralink,mt7620a-soc";`

Reply:
ok, thank you. I have replaced "apexera-netbot" with "apexera,netbot", also includes the board configs in platform.sh/ramips.sh/02_network

Reply:
I can not see any of the changes. Please confirm that you added them to your commit. Please don't mark the requested changes as resolved. They will be marked as resolved as soon as you change the code.